### PR TITLE
Update onedrive-office-hours.md

### PR DESCRIPTION
**Pull request by bigburrito21**
removed terms like support and customer so that this article is not the top search result for "OneDrive customer support"

### CODE REVIEW

File: SharePoint/SharePointOnline/onedrive-office-hours.md [hunk 1]
```diff
@@ -37,14 +37,10 @@ The OneDrive product and engineering team now offers monthly customer office hou
 
 - **Q&A:** we'll answer OneDrive questions and field your questions and suggestions. (~45 mins)
 
-**OneDrive customer office hours is not a support tool**
+**You may opt out of OneDrive office hours at anytime**
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/onedrive-office-hours.md:40]:
```suggestion
**You may opt out of OneDrive office hours at any time.**
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @bigburrito21, kindly commit the minor additional suggested change. Many thanks.

**Comment by bigburrito21** [comment]
Done

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by bigburrito21** [comment]
Do you know when the article changes will be published?
Microsoft OneDrive customer office hours - SharePoint in Microsoft 365 | Microsoft Learn<https://learn.microsoft.com/en-us/sharepoint/onedrive-office-hours>

**Comment by scanum** [comment]
@MikePlumleyMSFT
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**